### PR TITLE
May PR23: 12hr - Completed overall development of Governance subpage

### PR DESCRIPTION
**Pull request by gvirus21**
## Completed overall development of Governance subpage

- Improved Underline-gradient, 
- Adjusted font sizes
- Adjusted the Width of different components according to the screen sizes
- Added all the images
- Added all the required gradients to the page
- Improved Responsiveness of the page
- completed the governance subpage

### CODE REVIEW

**Comment by activate-glacier-instinct** [comment]
MINOR : Get started [mobile] 

I'd give this the same width as the section above. Needs bit more room to fix the `Get Started In` on one line

Screenshot from 2022-05-12 23-03-45

File: components/about-us/sub-pages/Governance/intro.tsx [hunk 1]
```diff
@@ -7,62 +7,91 @@ export default function GovernanceIntroComponent() {
...
+        position='absolute'
+        top={{ base: '0rem', sm: '0rem', md: '20rem' }}
+        left={{ base: '0', sm: '0', md: '2%', lg: '5%', xl: '10%' }}
+        bgGradient='linear(to-br, rgba(255, 16, 19, 0.3) 16.9%, rgba(140, 29, 207, 0.1) 88.02%)'
+        filter='auto'
+        blur='179px'
+        h='55rem'
+        w={{ base: '70%', md: '50rem' }}
+        borderRadius='full'
+        zIndices='hide'
```
Comment (activate-glacier-instinct) [inline components/about-us/sub-pages/Governance/intro.tsx:93]: QUESTION : should this be zIndex?

If not what is zIndices, what does it do here?

File: components/about-us/sub-pages/Governance/kickstart.tsx [hunk 2]
```diff
@@ -20,10 +20,12 @@ export default function GovernanceKickstartComponent() {
         spacing={{ base: 4, md: 6 }}
         py={{ base: 10, md: 18 }}
         mt={'4rem'}
+        position='relative'
+        zIndex='docked'
```
Comment (activate-glacier-instinct) [inline components/about-us/sub-pages/Governance/kickstart.tsx:24]: QUESTION : this is cool

Nice work taking advantage for the Chakra portals. Can you explain why you needed to this? Like what would happen if you didn't use this

**Comment by activate-glacier-instinct** [comment]
MINOR: overflow situation on Tablet bp

Screenshot from 2022-05-12 23-13-16

**Comment by activate-glacier-instinct** [comment]
MINOR : Quorum Requirements

These buttons should not be clickable

**Comment by activate-glacier-instinct** [comment]
MINOR : Quorum Requirements

These buttons should not be clickable

Screenshot from 2022-05-12 23-19-29

**Comment by activate-glacier-instinct** [comment]
MINOR : content 

Seasons links

Season 1 link - https://banklessdao.mirror.xyz/6i468DL59XhCAyfPoY3UA3h2uEDkkqWeiryuKkXOmBI

Season 2 link - https://banklessdao.mirror.xyz/Q3pMMBLblRHub5DW7wjsFhflqzlVLlBIhEjd9JwUjhk

Season 3 link - https://banklessdao.mirror.xyz/Ot8KDnNHZhN9-ZIp0hWUSblyPs83HfdMr7Kfp3HVoxw

**Comment by activate-glacier-instinct** [comment]
MINOR : buy BANK link 

https://www.coinbase.com/how-to-buy/bankless-dao

**Comment by activate-glacier-instinct** [comment]
MINOR: Buy Bank link

You might be able to reuse what I did on this PR for `The rug` 

That's on the About Us page

https://github.com/BanklessDAO/bankless-website/pull/20

In general, our links should be NextJS links with Chakra within

Full elements should be clickable - like you see on that PR on /about-us

**Comment by activate-glacier-instinct** [comment]
MINOR : Seasons link elements

Should have the same treatment as Buy Bank described above https://github.com/BanklessDAO/bankless-website/pull/23#issuecomment-1125476245

File: components/about-us/sub-pages/Governance/intro.tsx (hunk 1 above)
Comment (gvirus21) [reply components/about-us/sub-pages/Governance/intro.tsx:92]: Thanks for pointing this out, This was a mistake and didn't achieve anything. I intended to use zIndex here but it is pointless, as I am already moving Up ( in the z-axis ) both intro and kickstart section,

Fixed It

**Comment by activate-glacier-instinct** [comment]
Almost everything is done 

These are still there:
- zIndices
- https://github.com/BanklessDAO/bankless-website/pull/23#issuecomment-1125476245
- https://github.com/BanklessDAO/bankless-website/pull/23#issuecomment-1125476892

Got some screenshots for the links you'll see the markup for what is there and that is there in my branch of /read

The marking should be link first then button or p or whatever.

Screenshot from 2022-05-14 20-10-39
Screenshot from 2022-05-14 20-10-10

**Comment by CompositeFellow** [comment]
image
image
Looks good, one more thing.  Can you fix these bleed over issues at 320px width.

**Review by CompositeFellow — APPROVED** [review]
Bulk merge to evaluate our progress

**Pull request merged**